test/y2storage/guided_proposal_no_disks.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "spec_helper"
+require "y2storage"
+
+# stop rspec warning
+RSpec::Expectations.configuration.on_potential_false_positives = :nothing
```

Comment:
Why is this needed?

Moreover, I assume than placing it here actually affects the whole testsuite (or any test run after this). In that case, it's quite hidden here.

Reply:
Writing something like `to_not raise_error Y2Storage::Error` below looks less aggressive than disabling the warning (potentially) in the whole testsuite.

Reply:
Hm, I had expected it to affect only the single test file.

Without this rspec will not let you the error type you catch unspecified without a big warning.

Any better idea?

Reply:
It's a TypeError, actually. And this looked a bit too specific in a test to me.

Reply:
Well, if you want a completely generic check, I guess you can check for `Exception` (the root class for any exception)